04-gas/gas-section-four/gas-section-four.css

```diff
@@ -1,46 +1,107 @@
-.gas-section-four-container {
-  border: dotted 1px black;
+.gas-section-four-subcontainer,
+.gas-section-four-subcontainer,
+.gas-section-four-copy-container,
+.gas-section-four-title-container,
+.gas-section-four-paragraph-container,
+.gas-section-four-title,
+.gas-section-four-paragraph{
+  border: var(--temporary-border);
 }
 
-.gas-section-four-subcontainer {
+.gas-section-four-container {
   display: grid;
-   grid-template-columns: 1fr;
-   grid-template-areas: 
-  "gas-section-four-title-container"
-  "gas-section-four-paragraph";
-  justify-content: center;
-  row-gap: var(--items-gap-mobile);
+  grid-template-columns: 1fr;
   padding: var(--container-padding-mobile);
+  background-color: var( --level1-background-color);
 }
 
-.gas-section-four-subcontainer > * {
-  border: dotted 1px black;
+@media (min-width:600px){
+  .gas-section-four-container {
+    padding: var(--container-padding-desktop);
+  }
+}
+
+.gas-section-four-subcontainer {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--subcontainer-padding-mobile);
+  background-color: var( --level2-background-color);
 }
 
-@media (min-width:600px) {
-  .gas-section-four-subcontainer{
-   grid-template-columns: repeat(2,1fr);
-   grid-template-areas: 
-  "gas-section-four-title-container gas-section-four-paragraph";
+@media (min-width:600px){
+  .gas-section-four-container {
+    padding: var(--subcontainer-padding-desktop);
   }
 }
-  
-.gas-section-four-title-container{
-  grid-area: gas-section-four-title-container;
+
+.gas-section-four-copy-container {
   display: grid;
-  text-align: center;
-  align-content: center;
-  justify-content: center;
+  grid-template-columns: 1fr;
+  grid-template-areas: 
+  "gas-section-four-title-container"
+  "gas-section-four-paragraph-container";
+  align-content: start;
+  row-gap: var(--subcontainer-row-gap-mobile);
+  padding: var(--copy-container-padding-mobile);
+  background-color: var( --level3-background-color);
+}
+
+@media (min-width:600px){
+  .gas-section-four-copy-container{
+    grid-template-columns: repeat(2,1fr);
+    grid-template-areas: 
+    "gas-section-four-title-container gas-section-four-paragraph-container";
+    column-gap: var(--subcontainer-column-gap-desktop);
+  }
+}
+
+.gas-section-four-title-container {
+ grid-area: gas-section-four-title-container;
+ display: grid;
+ align-content: center;
```

Comment:
i guess here (IF?) the number and text have to be side by side you just do a grid with the first cell fitting its content and the second one stretching the remainder of the grid..

maybe you have a better idea of the design because you are spending more time with it, and how it works on the "even" number components 2 4 6 8 etc.

Reply:
i asked karleen about hero , so i follow same , bit i noticed now they are different, i need to ask , but she review section two and doesnt mention any changes. i will ask again